src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AttendanceController.java

```diff
@@ -48,11 +48,24 @@ public AttendanceDto postAttendance(
         return attendanceService.createAttendance(attendance);
     }
 
+    @PostMapping(value = "/bookings", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.CREATED)
+    @ApiOperation(value = "Create new attendance records for multiple offenders (This endpoint does not trigger IEP warnings)",
+            responseContainer = "Set",
+            notes = "Stores new attendance record for multiple offenders, posts attendance details back up to PNOMIS")
+    public Set<AttendanceDto> postAttendances(
```

Comment:
returning a simple collection can be a bit limiting for api evolvability, if you want to return additional metadata then it would be breaking change. (There's a bit where where it talks about this in the [api design standards](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#use-json))